cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/batch/BasicMapReduceContext.java

```diff
@@ -78,8 +81,8 @@
   private final File pluginArchive;
   private final Map<String, LocalizeResource> resourcesToLocalize;
 
-  private InputFormatProvider inputFormatProvider;
-
+  // key is input name, value is the MapperInput (configuration info) for that input
```

Comment:
Yea, it will be whatever `Input#getName` returns.
This will also be the same input exposed to the Mapper class.
So user can do `MapReduceTaskContext#getName()` from a mapper instance, to get the input name that this mapper is reading from.